tests/core_tests.py

```diff
@@ -161,24 +161,6 @@ def test_get_superset_tables_not_found(self):
         rv = self.client.get(uri)
         self.assertEqual(rv.status_code, 404)
 
-    def test_old_slice_json_endpoint(self):
```

Comment:
These tests are no longer applicable, as the backend is not able to infer 1) which endpoint to call 2) what the final request payload will look like (this is handled by `buildQuery.{j,t}s` in the plugin). Going forward all these tests will reside in the viz plugin, with the backend tests ensuring that valid chart data requests get valid responses.